### PR TITLE
created json file and added dependencies

### PR DESCRIPTION
**Pull request by harjothkhara**

### CODE REVIEW

File: data/helpers/actions-model.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+async function update(item, id) {
+    const editedAction = await db("actions")
+        .where({ id })
+        .update(item);
+    if (editedAction) {
+        const action = await findById(id);
+        return action;
+    }
+}
```
Comment (angel-torres) [inline data/helpers/actions-model.js]: ✔️

File: data/helpers/projects-model.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        const project = await findById(id);
+        project.completed === 1
+      ? (project.completed = true)
+      : (project.completed = false);
+        return project;
+    }
+}
+
+async function remove(id) {
+    const project = await findById(id);
```
Comment (angel-torres) [inline data/helpers/projects-model.js]: I find this cool article about async functions. 
https://dev.to/mywebstuff_hq/async-function-vs-a-function-that-returns-a-promise-3lpo

File: data/migrations/20190607082124_projects.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    table.string("notes", 256).notNullable();
+    table.boolean("completed");          
+  });
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema
+    .dropTableIfExists("actions")
+    .dropTableIfExists("projects")        
+};
```
Comment (angel-torres) [inline data/migrations/20190607082124_projects.js]: ✔️

File: routers/actionRouter.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        }
+      } catch (error) {
+        res.status(500).json({
+          message: `The action's information could not be modified: ${error}.`
+        });
+      }
+    }
+  });
+  
+  module.exports = router;
```
Comment (angel-torres) [inline routers/actionRouter.js]: ✔️

File: routers/projectRouter.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+            }
+        } catch(error) {
+            res.status(500).json({
+                message: `The project's information could not be modified: ${error}.`
+            });
+        }
+    }
+}); 
+
+module.exports = router;
```
Comment (angel-torres) [inline routers/projectRouter.js]: ✔️

**Review by angeltorgon — COMMENTED** [review]
## Minimum Viable Product Objectives
* [ ] Self-Study questions
* [x] `project` table with unique id, name, description, completed columns
* [x] `action` table with unique id, description, notes, completed columns
* [x] Build database and tables with `knex` migrations
* [x] Api endpoints: POST projects, POST actions, GET project by id 

## Stretch Objectives
* [x] Add remaining CRUD operations not required in MVP
* [ ] Add `context` support to app

# Project Manager Feedback
## Great
* You completed all MVP objectives! :heavy_check_mark:
* You completed stretch objectives! :heavy_check_mark:
Awesome work Harjoth!
## Requested Improvements
* Make sure you answer the self-study questions on `README.md` 
## Questions
* How comfortable to you feel setting up a server from scratch?
## Rating {3}
:star: :star: :star:

**Pull request merged**